app/assets/javascripts/models/beta/story.js

```diff
@@ -207,6 +207,22 @@ export const setLoadingValue = (story, loading) => ({
   loading
 });
 
+export const cloneStory = (story) => {
+  const clonedStory = {
+    ...story,
+    id: null,
+    state: status.UNSCHEDULED,
+    _isDirty: true,
+    collapsed: false,
+    _editing: {}
```

Comment:
Can we set it to `null` instead of an empty object so we don't waste memory here?